src/manager.cpp

```diff
@@ -131,7 +132,10 @@ void Manager::syncData(const config::DataSyncConfig& dataSyncCfg)
 {
     using namespace std::string_literals;
     std::string syncCmd{"rsync --archive --compress"};
-
+    if (!std::filesystem::exists(dataSyncCfg._path))
```

Comment:
I believe this check should not be added here, as it's a common API used for various sync types. Some of these may try to delete a file on a sibling BMC if it has been removed from the local BMC.